packages/contracts/contracts/utils/Ratio.sol

```diff
@@ -11,10 +11,10 @@ uint256 constant RATIO_BASE = 10 ** 6;
 error RatioOutOfBounds(uint256 limit, uint256 actual);
 
 /// @notice Applies a ratio to a value and ceils the remainder.
-/// @param _value The value to which the ratio is applied .
-/// @param _ratio The ratio.
+/// @param _value The value to which the ratio is applied.
+/// @param _ratio The ratio that must be in the interval `[0, 10**6]`.
 /// @return result The resulting value.
-function applyRatioCeiled(uint256 _value, uint256 _ratio) pure returns (uint256 result) {
+function _applyRatioCeiled(uint256 _value, uint256 _ratio) pure returns (uint256 result) {
```

Comment:
Why don't we leave these function names as they were?  We could simply use `testApplyRatioCeiled()` on the new test contract, relaying to `applyRatioCeiled()`

(suggestion, not important)

Reply:
File-level functions will be included as internal functions in the contract. Our convention is to add an `_` as a prefix so I thought it is better to make clear that this is not a public function.
We do the same with `_auth` from `auth.sol`